research/src/pages/component-spec-template.mdx

```diff
@@ -0,0 +1,213 @@
+> This document represents a component specification template. Its purpose
+> is to provide a point-in-time guide as to how a component should be
+> authored, in light of common anatomies, behaviors, and accessibility.
+> The primary audience is a developer authoring a given component.
+
+# {ComponentName} component specification
+
+## Overview
+
+*The name of the component, along with a high-level description.*
+
+### Background
+
+*Relevant historical or background information, related existing issues, etc.*
+
+### Use Cases
+
+*Primary use cases for this component.*
+
+### Non-goals
+
+*A list of use cases, features, or functionality which are **not** goals for the component.*
+  
+### Features
+
+*A list of the key features unique to this component.*
+
+### Risks and Challenges
+
+*Notable risks or challenges associated with implementing or shipping the component.*
+
+### Prior Art/Examples
+
+*Screenshots and/or links to existing, canonical, or exemplary implementations of the component.*
+
+---
+
+## Design
+
+*Describe the design of the component, thinking through several perspectives:*
+
+- *A customer using the component on a web page.*
+- *A developer building an app with the component and interacting through HTML/CSS/JavaScript.*
+- *A designer customizing the component.*
+
+### API
+
+*Outline the key elements of the component's public API surface, taking into consideration conformance guidelines. Make sure to discuss differences and rationalizations. Consider high and low-level APIs. Attempt to design a powerful and extensible low-level API with a high-level API for developer/designer ergonomics and simplicity.*
+
+#### Tag Name
+
+*The tag that would be used in HTML. e.g. `<my-slider></my-slider>`* 
```

Comment:
@EisenbergEffect - I'm wondering if we need this for the initial specification. This seems more along the lines a team implementing this spec would care about, whereas the intent here is to document the overall pattern for alignment. I think we can probably remove this as it's going to be specific to the implementation (as WC's require).

Reply:
Makes sense. I'll remove this section. A general name would be included in the spec title and overview, so I think that's sufficient for the spec's purpose.